src/test-utilities/react-testing.tsx

```diff
@@ -1,80 +1,26 @@
 import React from 'react';
 import {createMount, mount} from '@shopify/react-testing';
-import {createThemeContext} from '../utilities/theme';
-import {ScrollLockManager} from '../utilities/scroll-lock-manager';
-import {StickyManager} from '../utilities/sticky-manager';
-import {createAppBridge} from '../utilities/app-bridge';
-import {I18n} from '../utilities/i18n';
 import translations from '../../locales/en.json';
-import {merge} from '../utilities/merge';
-import {PolarisTestProvider} from './PolarisTestProvider';
-import {WithProvidersOptions, WithProvidersContext} from './types';
+
+import {
+  PolarisTestProvider,
+  WithPolarisTestProviderOptions,
+} from './PolarisTestProvider';
 
 export {mount};
 
-export const mountWithContext = createMount<
-  WithProvidersOptions,
-  WithProvidersContext
+export const mountWithApp = createMount<
```

Comment:
I agree that `mountWithContext` may have not been descriptive enough, but I'm not sold on `app` since that not entirely what polaris is. How about `mountWithPolaris`?

Reply:
I was thinking drawing a parallel with `<AppProvider>`,  as mountWithApp is the equivalent of having an element inside `<AppProvider><Frame>BLAH</Frame></AppProvider>`.

mountWithPolaris would be better if we called that PolarisProvider, but alas that horse has bolted.